src/borg/key.py

```diff
@@ -18,14 +18,20 @@
 from .helpers import Chunk
 from .helpers import Error, IntegrityError
 from .helpers import yes
-from .helpers import get_keys_dir
+from .helpers import get_keys_dir, get_key_nonces_dir
 from .helpers import bin_to_hex
 from .helpers import CompressionDecider2, CompressionSpec
 from .item import Key, EncryptedKey
+from .platform import SaveFile
+from .remote import InvalidRPCMethod
 
 
 PREFIX = b'\0' * 8
+NONCE_SPACE_RESERVATION = 1000
```

Comment:
Yes, i think we want this bigger. But going all the way to 4GB seems to be quite much. The way this currently works is that even a delete will consume 4GB of nonce space. Not sure that’s a problem though. 2**64 is a big number. Still gives 2**32 deletes till the nonce space is exhausted.